web-api/src/main/java/com/graphhopper/util/CustomModel.java

```diff
@@ -28,8 +28,7 @@ public class CustomModel {
 
     public static final String KEY = "custom_model";
 
-    // e.g. 70 means that the time costs are 25€/hour and for the distance 0.5€/km (for trucks this is usually larger)
-    static double DEFAULT_DISTANCE_INFLUENCE = 70;
```

Comment:
Where is the server-side default now? Is it still 70?

Reply:
It has been bugging me for a while now that our "fastest" routing doesn't really pick the fastest route by default.

Reply:
Maybe mention this in the changelog as well: Server-side profiles with custom weighting now use distance_influence: 0 by default (previously it was 70).

Reply:
As someone who expected the fastest route when using a CustomModel, the default value for "distance_influence" caught me a bit off guard. It is not obvious at the first moment that here the length of the route is included in the weighting by default.

I think that this is not clear to most who use this feature.

Reply:
Ok, so far I wasn't aware that the expectation of an empty CustomModel is that it yields the fastest route. But anyway, I agree that 0 is probably the better default since 70 is quite arbitrary.